specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json

```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip/checkRouting/{targetNumber}": {
+      "get": {
+        "tags": [
+          "NumberRoutingValidation"
+        ],
+        "summary": "Gets the list of routes matching the target phone number, ordered by priority.",
+        "operationId": "CheckRoutingForNumber",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "targetNumber",
```

Comment:
nit: Is "targetNumber" a domain specific name? I find the adjectives "source" and "destination" more natural in the context of a connection / path etc.

Reply:
In the case of PSTN calling I find the expression "target phone number" to be more natural in English than "destination phone number". I'd advocate staying with "target"